### PR TITLE
Add the missing field for the delivery methods

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request adds the missing field for the delivery methods:
- Flat Rate
- Table Rates

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/shipping/shipping-table-rate.html
https://docs.magento.com/user-guide/shipping/shipping-flat-rate.html

### CODE REVIEW

File: src/shipping/shipping-table-rate.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ The first step is to complete the default settings for table rates. You can comp
    | All Allowed Countries | Customers from all [countries]({% link stores/country-options.md %}) specified in your store configuration can use this delivery method. |
    | Specific Countries | When you choose this option, the _Ship to Specific Countries_ list appears. Select each country in the list where this delivery method can be used. |
 
+1. Set **Show Method if Not Applicable** to `Yes` if you want to show Table Rates all time
```
Comment (BarnyShergold) [inline src/shipping/shipping-table-rate.md:75]:
```suggestion
1. Set **Show Method if Not Applicable** to `Yes` if you want to show Table Rates all the time
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Nice clarification - one small change needed

File: src/shipping/shipping-table-rate.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/shipping/shipping-table-rate.md]: Done, please check) Thanks ☺️

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**